tests/test_cloudpy.py

```diff
@@ -1,18 +1,26 @@
 # -*- coding: utf-8 -*-
+import datetime
 import json
 import os
 import unittest
 
 from xoxzo.cloudpy import XoxzoClient
 
 
-class TestXoxzoClient(unittest.TestCase):
+class TestXoxzoClientTestCase(unittest.TestCase):
+    def getenv_with_none_check(self, env):
+        val = os.environ.get(env)
+        if val == None:
+            raise Exception("Environment variable %s must be set" % env)
+        return val
+
```

Comment:
I think this is more kind for user since it will show different error message when
1. User forget to set the environment variable.
2. User set the wrong key in the environment variable